plasmapy/physics/parameters.py

```diff
@@ -5,8 +5,8 @@
 from astropy.units import (UnitConversionError, UnitsError, quantity_input,
                            Quantity)
 
-from ..constants import (m_p, m_e, c, mu0, k_B, e, eps0, pi, ion_mass,
-                         charge_state)
+from ..constants import (m_p, m_e, c, mu0, k_B, e, eps0, pi)
+from ..atomic import (ion_mass, charge_state)
```

Comment:
Come to think about it that's a funny import. Are the `( )` in thee to keep things working without `\` as it used to go multiline, like in line 5 up there? If so, that's cool, never thought about it!

Reply:
Yep, the [parentheses help with making multiline imports cleaner](https://www.python.org/dev/peps/pep-0328/).  Even on a single line, I sometimes use them since I personally find the code to be slightly more readable that way.

Reply:
Cool, today I learned. Thanks!